docs/docsite/rst/filter_guide-abstract_informations-types-reveal_ansible_type.rst

```diff
@@ -0,0 +1,144 @@
+..
+  Copyright (c) Ansible Project
+  GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+  SPDX-License-Identifier: GPL-3.0-or-later
+
+Filter reveal_ansible_type
+""""""""""""""""""""""""""
+
+Use the filter :ansplugin:`community.general.reveal_ansible_type#filter` if you want to get the type of Ansible data.
+
+.. note:: The output of the examples in this section use the YAML callback plugin. Quoting: "Ansible output that can be quite a bit easier to read than the default JSON formatting." See :ansplugin:`the documentation for the community.general.yaml callback plugin <community.general.yaml#callback>`.
+
+.. versionadded:: 9.2.0
+
+**Substitution converts str to AnsibleUnicode**
+
+* String. AnsibleUnicode.
```

Comment:
What does this mean? (I know what it means, but regular users do not.)

Reply:
I'll improve all the comments.